src/amqp-ts.ts

```diff
@@ -425,7 +420,7 @@ export class Message {
     if (destination.initialized.isFulfilled()) {
       return sendMessage();
     } else {
-      return (<Promise<any>>destination.initialized).then(sendMessage);
+      return Promise.reject(new Error("Connection is not ready."));
```

Comment:
Yes it does wait. What it means is that now, it will thrown an error that will be caught by the rabbitMQClient and retried through SQS.

Reply:
ahh coolio, SQS will handle it for us 😎 

It could be nice to add the tests you made in the description to make sure SunCo apps are still happy with those changes.